backend/src/test/resources/testResultUpload/test-results-upload-valid-different-results.csv

```diff
@@ -0,0 +1,7 @@
+1234,Doe,Jane,,123 Main St,,Birmingham,AL,35226,Jefferson,205-999-2800,1/20/1962,F,White,Not Hispanic or Latino,,,123,ID NOW,94534-5,Positive,12/20/2021 14:00,12/21/2021 14:00,12/22/2021 14:00,12/23/2021 14:00,12/24/2021 14:00,Nasal Swab,1013012657,Smith MD,John,,400 Main Street,,Birmingham,AL,35228,205-888-2000,01D1058442,My Testing Lab,300 North Street,,Birmingham,AL,35228,205-888-2000,N,N,N,,N,,N,N,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,Test Comment,,F
+2345,Doe,Jayne,,123 Main St,,Birmingham,AL,35226,Jefferson,205-999-2800,1/20/1962,F,White,Not Hispanic or Latino,,,234,ID NOW,94534-5,Negative,12/20/2021 14:00,12/21/2021 14:00,12/22/2021 14:00,12/23/2021 14:00,12/24/2021 14:00,Nasal Swab,1013012657,Smith MD,John,,400 Main Street,,Birmingham,AL,35228,205-888-2000,01D1058442,My Testing Lab,300 North Street,,Birmingham,AL,35228,205-888-2000,N,N,N,,N,,N,N,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,Test Comment,,F
+3456,Doe,Jan,,123 Main St,,Birmingham,AL,35226,Jefferson,205-999-2800,1/20/1962,F,White,Not Hispanic or Latino,,,345,ID NOW,94534-5,Detected,12/20/2021 14:00,12/21/2021 14:00,12/22/2021 14:00,12/23/2021 14:00,12/24/2021 14:00,Nasal Swab,1013012657,Smith MD,John,,400 Main Street,,Birmingham,AL,35228,205-888-2000,01D1058442,My Testing Lab,300 North Street,,Birmingham,AL,35228,205-888-2000,N,N,N,,N,,N,N,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,Test Comment,,F
+4567,Doe,Jayn,,123 Main St,,Birmingham,AL,35226,Jefferson,205-999-2800,1/20/1962,F,White,Not Hispanic or Latino,,,456,ID NOW,94534-5,Not Detected,12/20/2021 14:00,12/21/2021 14:00,12/22/2021 14:00,12/23/2021 14:00,12/24/2021 14:00,Nasal Swab,1013012657,Smith MD,John,,400 Main Street,,Birmingham,AL,35228,205-888-2000,01D1058442,My Testing Lab,300 North Street,,Birmingham,AL,35228,205-888-2000,N,N,N,,N,,N,N,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,Test Comment,,F
+4567,Doe,Janey,,123 Main St,,Birmingham,AL,35226,Jefferson,205-999-2800,1/20/1962,F,White,Not Hispanic or Latino,,,567,ID NOW,94534-5,Invalid Result,12/20/2021 14:00,12/21/2021 14:00,12/22/2021 14:00,12/23/2021 14:00,12/24/2021 14:00,Nasal Swab,1013012657,Smith MD,John,,400 Main Street,,Birmingham,AL,35228,205-888-2000,01D1058442,My Testing Lab,300 North Street,,Birmingham,AL,35228,205-888-2000,N,N,N,,N,,N,N,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,Test Comment,,F
+4567,Doe,Janey,,123 Main St,,Birmingham,AL,35226,Jefferson,205-999-2800,1/20/1962,F,White,Not Hispanic or Latino,,,678,ID NOW,94534-5,260373001,12/20/2021 14:00,12/21/2021 14:00,12/22/2021 14:00,12/23/2021 14:00,12/24/2021 14:00,Nasal Swab,1013012657,Smith MD,John,,400 Main Street,,Birmingham,AL,35228,205-888-2000,01D1058442,My Testing Lab,300 North Street,,Birmingham,AL,35228,205-888-2000,N,N,N,,N,,N,N,My Urgent Care,400 Main Street,Suite 100,Birmingham,AL,35228,205-888-2000,Test Comment,,F
```

Comment:
This file contains all the accepted values for test results 
- Positive
- Negative
- Detected
- Not Detected
- Invalid Result
- snomed value (260373001)